authappliance/pi-appliance

```diff
@@ -157,8 +157,7 @@ class WebserverMenu(object):
 
 class Peer(object):
 
-    files = ["/etc/privacyidea/enckey", "/etc/privacyidea/pi.cfg",
-             "/etc/privacyidea/logging.cfg",
+    files = ["/etc/privacyidea/enckey",
```

Comment:
... oh, but ``logging.cfg`` should probably still be copied over to Server 2, I guess?